src/walletSdk/Auth/WalletSigner.ts

```diff
@@ -28,3 +37,42 @@ export const DefaultSigner: WalletSigner = {
     );
   },
 };
+
+export class DomainSigner implements WalletSigner {
+  private url: string;
+  private client: AxiosInstance;
+  private headers: HttpHeaders;
+  constructor(url: string, headers: HttpHeaders) {
```

Comment:
Can you add a js doc that clarifies that headers is intended to use for authentication?

Reply:
👍  https://github.com/stellar/typescript-wallet-sdk/pull/53/commits/02c1922e84ae55360c9fbc248595e21b03098bd1